vars/pyenv.groovy

```diff
@@ -5,10 +5,21 @@ import com.ableton.VirtualEnv
 /**
  * Create a new {@link com.ableton.VirtualEnv} instance using pyenv.
  * @param pythonVersion Python version, as given by pyenv versions --list.
- * @param pyenvRoot Path of the pyenv installation. If {@code null}, then the pyenv root
- *        will be detected from the environment in {@code PYENV_ROOT}.
```

Comment:
Isn't this still the case? Shouldn't we leave the documentation in?

Reply:
I guess it's the case that the pyenv root needs to be specified somehow, but it's not the case that this parameter exists anymore. :smile: Anyways, you're right, we should say _something_. Fixed in b923a9f.